### DISCUSSION TITLE
How do i correctly save pokemon?

### DISCUSSION THREAD

**Discussion by Mrgamer2240** [kwsch/PKHeX#4313] [General]
I watched some tutorials on how to use Pkhex but for reasons they couldn't cover the entire saving process. I'm stuck on actually getting the pokemon in my game, I have a pathway it's SD card/JkSV/Pokemon Shield/ "Playername".I replaced the "main" folder with the modified one but the pokemon don't show.

**Comment by Lusamine** [comment]
After you edit your Pokemon, you need to right click and set it in a box. Then File and Export your save to somewhere on your computer before you copy to SD card.

**Comment by Mrgamer2240** [comment]
To Confirm Where on the SD do I save it, because my pathway is not working.

**Reply by kwsch** [reply]
"not working" is too vague. If your sd card is locked, then unlock it. Try saving it elsewhere and copying it manually. 

Be sure to use jksv to reimport your edited save file once it's back on your 3DS.

**Reply by Lusamine** [reply]
If you're asking how to use JKSV to manage saves, Project Pokemon has a guide here: https://projectpokemon.org/home/tutorials/save-editing/managing-switch-saves/using-jksv-r88/

**Comment by Mrgamer2240** [comment]
Do you suggest I use "USB File Transfer" or manually put the SD card in the pc?

**Reply by kwsch** [reply]
Either method works. 

This isn't really related to PKHeX. All the program does is edit files you give it, regardless of origin. Once you are done editing the file, it's up to you to move/import it with whatever tools you're already using to bring the file into the program.

**Comment by Mrgamer2240** [comment]
Every time I try to edit the save I load the game to no avail, the pokemon from pkhex never appear, I don't know if this is because my Sd card is not functioning or maybe my switch it's self, unrelated but the only th8ng I can seem to is edit trainer's nothing else I might  be missing a huge step so here a pathway to confirm weather or not I'm doing this correctly. SD card/JkSV/Pokemon Shield/ "Playername" then replace "main" with the modified "main" version. I launch the game after this but nothing happens, Pkhex also tells me that my SD card is write protected however I checked twice and it's not.

**Reply by Lusamine** [reply]
This is incorrect. You need to read the JKSV guide and import the new save after you have moved it to your SD card. Again, this isn't a problem with or question about PKHeX at all, but a question about how to use homebrew/ your computer.

You should not be trying to edit your save while it's in your SD card. Export the save with JKSV, move to PC, edit it and export to somewhere on your PC, move it back to SD card, import with JKSV. Review the guide I linked if it's unclear.

**Comment by Mrgamer2240** [comment]
Update: when I open a file Pkhex says it's write protected,  I believe this is causing my pokemon not to go through I already checked for any errors but my pc still says it's not protected while Pkhex says it is.

**Reply by kwsch** [reply]
No, the program is literally telling you that your file is write protected, because it is, as we've previously told you.

When you are done modifying your save data, save it ANYWHERE ELSE then move it manually. Your 3DS sd card is locked, and nothing can write to it until you unlock it. This is not an issue with the program.
